go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/pingcap/log v0.0.0-20200117041106-d28c14d3b1cd
 	github.com/pingcap/tidb-tools v4.0.0-beta.1.0.20200306103835-530c669f7112+incompatible
 	github.com/pkg/errors v0.9.1
+	github.com/soheilhy/cmux v0.1.4
```

Comment:
why?

Reply:
+1

Reply:
after I ran`make test`, this go mod will be auto generated, should I remove it?

Reply:
No, this is added because I forget to add cmux in last PR.